datapoint-forward/test/Test/DataPoint/Forward/Types.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+
+module Test.DataPoint.Forward.Types
+  ( NodeInfo (..)
+  , BlockchainStatus (..)
+  ) where
+
+import qualified Data.Aeson as A
+import           Data.Text (Text)
+import           GHC.Generics
+
+data NodeInfo = NodeInfo
```

Comment:
@denisshevchenko, can we rename it so it doesn't cause "grep conflicts" with the real `NodeInfo`?

Reply:
Yes, I'll do it.